code/switch.js

```diff
@@ -0,0 +1,167 @@
+let pastaType
+let saladType
+
+switch (foodTypeSelected) {
+    case "Pizza":
+        pizzaType = prompt(`Great! We have these to choose from:
+        1. Margherita 11€
+        2. Verdure    12€
+        3. Surprise   14€`
+        );
+        break;
+    case "Pasta":
+        pastaType = prompt(`Great! We have these to choose from:
+        1. Arrabbiata   12€
+        2. Casio e Pepe 10€
+        3. Alfredo      9€`
+        );
+        break;
+    case "Salad":
+        saladType = prompt(`Great! We have these to choose from:
+        1. Panzanella   6€
+        2. Caprese      7€
+        3. Giardiniera  8€`
+        );
+        break;
+    default: alert(`Sorry, we did not catch that. Type a number.`);
+            process.exit(1);
+}
+//VALUES FOR EACH FOOD TYPE
+
+let foodPrice
+let foodOrder
+
+switch (foodTypeSelected) {
+    case "Pizza":
+        switch (pizzaType){
+            case "1":
+                foodOrder = "Margherita";
+                foodPrice = 11;
+                break;
+            case "2":
+                foodOrder = "Verdure";
+                foodPrice = 12;
+                break;
+            case "3":
+                foodOrder = "Surprise";
+                foodPrice = 14;
+                break;
+            default: alert("Invalid choice. Refresh and try again.");
+            break;
+        }
+        break;
+    case "Pasta":
+        switch (pastaType){
+            case "1":
+                foodOrder = "Arrabbiata";
+                foodPrice = 12;
+                break;
+            case "2":
+                foodOrder = "Casio e Pepe";
+                foodPrice = 10;
+                break;
+            case "3":
+                foodOrder = "Alfredo";
+                foodPrice = 9;
+                break;
+            default: alert("Invalid choice. Refresh and try again.");
+            break;
+        }
+        break;
+    case "Salad":
+        switch (saladType){
+            case "1":
+                foodOrder = "Panzanella";
+                foodPrice = 6;
+                break;
+            case "2":
+                foodOrder = "Caprese";
+                foodPrice = 7;
+                break;
+            case "3":
+                foodOrder = "Giardiniera";
+                foodPrice = 8;
+                break;
+            default: alert("Invalid choice. Refresh and try again.");
+                process.exit(1);
+        }
+        break;
+            
+}
+// ANSWER USER
+alert(`You have ordered the ${foodOrder}! That'll be ${foodPrice} €, thank you!`)
+
+//REST OF THE CODE IS SAME AS IN SCRIPT.JS !
+
+  // Step 4 - Age
+  let babyPrice = (foodPrice/2)
+  let age = parseInt(prompt(`I forgot to ask, how old are you?
+  If under 16, you will get a smaller portion for half price! \nPlease type in your age!`));
+  if (age <= 16 ) { 
+    alert(`Oh, there we go, a smaller plate for only ${babyPrice}€ !`)}
+  else { alert(`All in order! A big portion for ${foodPrice} €.`)}
```

Comment:
Noticed that this if/else statement still accepts a string as prompt input. If the user enters a non-number, eg "hey", the alert will say "All in order" A big portion for ${foodPrice}€." And when the user clicks onwards, it will then say "So sorry we got it wrong, please try again" (that is invoked by line 159). 
To solve this, the if/else statement here could contain and additional "else if" eg: 

```
if (age <= 16 ) { 
    alert(`Oh, there we go, a smaller plate for only ${babyPrice}€ !`)}
  else if (>16) { alert(`All in order! A big portion for ${foodPrice} €.`)}
else { alert (`So sorry we got I wrong, please try again!`)}
```